build/build_image.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+# Copyright 2021 Chaos Mesh Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+import argparse
+import subprocess
+import pathlib
+
+import utils
+import common
+
+def get_image_env(name, env, default):
+    default_env = os.getenv("IMAGE_" + env, default)
+    if default_env == "":
+        default_env = default
```

Comment:
Why do we need to set default value again after `os.getenv(name, default)`?

Reply:
Because the environment variable can be set to an empty string. For example, if the `IMAGE_TAG` is not set, but passed as `IMAGE_TAG=${IMAGE_TAG}` in the `Makefile`, then even though we get the variable through `os.getenv("IMAGE_TAG", "latest")`, we will get `""`.

Reply:
I get it, but it's a mechanism to handle the lack of default values in `Makefile`, right? Here is another question, we maintain default values in two places at the same time, I think it's maybe not a good way, because we need to update both two places if we want to modify a default value. IMO, the `Makefile` need to ensure its correction and become a reliable entrance. 🙈

Reply:
Good! You hit the goal! Is it a good idea to only specify default values in python script? I will try to migrate the environment variable settings in the `Makefile` to the script.

Reply:
I think we need to add enough documents if we migrate env to script, refer to https://github.com/kubernetes/kubernetes/blob/master/build/root/Makefile#L210 🤔

Reply:
@iguoyr I attempted to only set default value inside the `Makefile`, but it seems too complicated (as there are still some variables only depended by tasks who don't need python script).

I will remove default value in python script, and only set default value inside the `Makefile` to see whether it will work.